cmd/juju/caas/add.go

```diff
@@ -192,6 +197,7 @@ func (c *AddCAASCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.StringVar(&c.clusterName, "cluster-name", "", "Specify the k8s cluster to import")
 	f.StringVar(&c.contextName, "context-name", "", "Specify the k8s context to import")
 	f.StringVar(&c.hostCloudRegion, "region", "", "kubernetes cluster cloud and/or region")
+	f.StringVar(&c.cloud, "cloud", "", "kubernetes cluster cloud and/or region")
```

Comment:
"kubernetes cluster cloud type"

Reply:
as discussed, the cloud could be name or type.